src/options/views/edit/values.vue

```diff
@@ -168,6 +180,19 @@ export default {
     onCancel() {
       this.current = null;
     },
+    onChange() {
+      this.reparseJson(this.current.value);
```

Comment:
I'm not sure if this would work well when the value is very large. I'll do some tests later.

Reply:
Fixed by disabling the automatic check once it exceeds 10ms.